examples/linear_model/plot_poisson_regression_non_normal_loss.py

```diff
@@ -393,11 +393,11 @@ def _mean_frequency_by_risk_group(y_true, y_pred, sample_weight=None,
 #
 # To compare the 3 models within this perspective, one can plot the fraction of
 # the number of claims vs the fraction of exposure for test samples ordered by
-# the model predictions, from safest to riskiest  according to each model:
```

Comment:
is this change related?

Reply:
Nope, got an issue with my last merge from master, just pushed the fix :)